platforms/ios/demo/src/MapViewController.m

```diff
@@ -18,9 +18,17 @@ - (void)addAlert:(NSString *)message withTitle:(NSString *)title;
 
 @implementation MapViewControllerDelegate
 
+- (void)mapView:(TGMapViewController *)view didCaptureScreenshot:(UIImage *)screenshot
+{
+    NSArray* paths = NSSearchPathForDirectoriesInDomains(NSDocumentDirectory, NSUserDomainMask, YES);
```

Comment:
We actually don't really need that, it was used only for testing in the demo app, I will remove it and rebase.